app/services/message/extract_keyword.rb

```diff
@@ -17,15 +17,14 @@ def call
                   "unavailable" => "Unavailable",
                   "no"          => "Unavailable",
                   "upcoming"    => "Upcoming",
-                  "info"        => "Info"}
+                  "info"        => "Info",
+                  "arrive"      => "Arrive",
+                  "depart"      => "Depart"}
     # TODO Additional keywords => maybe rcv received ack
     # TODO tasks task roster (who's assigned to event)
     # TODO schedule (What I'm scheduled for)
     # TODO onduty on-duty for who's on duty right now. & signin signout
     # TODO "crew id_code" to allow supervisor to see who has responded.
-    if white_list.keys.include? first_word
-      return white_list[first_word]
-    end
-    return nil
+    return white_list[first_word]
```

Comment:
Not strictly necessary for this pr, but I think this is more concise and perhaps faster. Tests continue to pass.